hack/remove-calico-policy/iptables-remover-ds.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: calico-remove-iptables
+  namespace: kube-system
+  labels:
+    k8s-app: calico-remove-iptables
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: calico-remove-iptables
+    spec:
+      hostNetwork: true
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: projectcalico.org/ds-ready
+                  operator: NotIn
+                  values:
+                  - "true"
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+      containers:
+        - name: calico-remove-iptables
+          image: calico/iptables-remover
```

Comment:
why is it even a step in the process? I'm just thinking it looks different from all the other Calico components where we just include the Dockerfile and host it on dockerhub but building the image isn't a step.